opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -286,6 +354,8 @@ def __init__(self, name: str = "") -> None:
         if name:
             slot_name = "{}.current_span".format(name)
         self._current_span_slot = Context.register_slot(slot_name)
+        self._active_span_processor = MultiSpanProcessor()
+        self._lock = threading.Lock()
```

Comment:
Actually, this _lock is unused now, isn't it?

Reply:
You're right.